deploy/olm-catalog/ocs-operator/manifests/ocs-operator.clusterserviceversion.yaml

```diff
@@ -1245,7 +1250,7 @@ spec:
                 - name: OPERATOR_NAME
                   value: ocs-operator
                 - name: ROOK_CEPH_IMAGE
-                  value: rook/ceph:v1.3.2
+                  value: rook/ceph:v1.3.4
```

Comment:
If you use this tag `rook/ceph:v1.3.4-4.gb97be10` for the image you will pick up the latest change for the default `data` field on the device sets.

Reply:
ack.